x-pack/plugins/cases/public/components/user_actions/comment/comment.tsx

```diff
@@ -17,6 +17,24 @@ import { createAlertAttachmentUserActionBuilder } from './alert';
 import { createActionAttachmentUserActionBuilder } from './actions';
 
 const getUpdateLabelTitle = () => `${i18n.EDITED_FIELD} ${i18n.COMMENT.toLowerCase()}`;
+const getDeleteLabelTitle = () => `${i18n.REMOVED_FIELD} ${i18n.COMMENT.toLowerCase()}`;
```

Comment:
nitpick: considering you will unlikely reuse this function, you can simply inline the string concatenation in the line 28 where you use it.